core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -184,7 +184,7 @@ <h3>Subtopics</h3>
             <span>Name</span>
             <span class="number-skills-header"># of Questions</span>
           </div>
-          <div class="subtopic-name-header" ng-class="{'skills-valid': !uncategorizedSkillSummaries.length, 'skills-invalid': uncategorizedSkillSummaries.length}">
+          <div class="subtopic-name-header" ng-if="uncategorizedSkillSummaries.length" ng-class="{'skills-invalid': uncategorizedSkillSummaries.length}">
```

Comment:
Hm, if we are only going to show this when "uncategorizedSkillSummaries.length" is truthy, then we don't need to dynamically add / remove the class `skills-invalid`, right? We can use it as a regular class that will be applied all the time.

Reply:
Yup, True. Made the changes.